### PR TITLE
Fix a typo: inforamtion -> information

### PR DESCRIPTION
**Pull request by tiraboschi**
Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**What this PR does / why we need it**:
Fix a typo: inforamtion -> information

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/handler-launcher-com/notify/info/info.pb.go [hunk 1]
```diff
@@ -13,15 +13,12 @@ It has these top-level messages:
 */
 package info
 
-import (
-	fmt "fmt"
-
-	proto "github.com/golang/protobuf/proto"
-
-	math "math"
+import proto "github.com/golang/protobuf/proto"
```
Comment (oshoval) [inline pkg/handler-launcher-com/notify/info/info.pb.go:16]: there are some files changed which are not related to the PR, 
(all the ones with the import updating)
also they update it in a weird way, maybe you have some tool locally that cause it ?

**Review by oshoval — COMMENTED** [review]
Thanks, please see inline

File: pkg/handler-launcher-com/notify/info/info.pb.go (hunk 1 above)
Comment (maya-r) [reply pkg/handler-launcher-com/notify/info/info.pb.go:16]: perhaps generated with something other than `make generate`

Comment (davidvossel) [reply pkg/handler-launcher-com/notify/info/info.pb.go:16]: try running `make && make generate` and committing the results to see if that resolves the issue.

Comment (tiraboschi) [reply pkg/handler-launcher-com/notify/info/info.pb.go]: done

**Comment by dankenigsberg** [comment]
/lgtm

imho the release note would be unhelpful to users - it does not tell where the typo is fixed. I'd drop it.

**Comment by tiraboschi** [comment]
@davidvossel ping

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

Thanks!
